### ISSUE TITLE
plugin test script is failing - CI badges missing

### ISSUE THREAD

**Issue by janpieper** [asdf-vm/asdf-plugins#943]
### Describe the Bug

The CI badges in the README have been removed by 15dadb14e1daca61fdae4c4204a5d24c138e6fee, but `scripts/test_plugin.bash` is still looking for them, which causes the CI to fail.

For example, see #942.

### Steps to Reproduce

Just examples:

```
$ ./scripts/test_plugin.bash --file plugins/elixir
FAIL: Expected plugin CI badge SVG to be passing but it was not:
```

```
$ ./scripts/test_plugin.bash --file plugins/erlang
FAIL: Expected plugin CI badge SVG to be passing but it was not:
```

### Expected Behaviour

The `scripts/test_plugin.bash` should not check the badge anymore.

### Actual Behaviour

The `scripts/test_plugin.bash` is failing, because the CI badges are missing.

### Environment

:shrug:

```shell
$ asdf-info
asdf-info: command not found
```

**Comment by jthegedus** [comment]
>is still looking for them, which causes the CI to fail.

Too right! I was going to make some further updates to this but GitHub went down :upside_down_face: 

Will look to get this fixed shortly.

**Comment by jthegedus** [comment]
Ah, you beat me too it! thanks!

**Comment by jthegedus** [comment]
Closed in #944
